src/asset-swapper/types.ts

```diff
@@ -370,7 +370,7 @@ export enum SwapQuoteConsumerError {
 export enum SwapQuoterError {
     NoEtherTokenContractFound = 'NO_ETHER_TOKEN_CONTRACT_FOUND',
     StandardRelayerApiError = 'STANDARD_RELAYER_API_ERROR',
-    InsufficientAssetLiquidity = 'INSUFFICIENT_ASSET_LIQUIDITY',
+    InsufficientAssetLiquidity = 'We are not able to fulfill an order for this token pair at the requested amount due to a lack of liquidity',
```

Comment:
We should keep the enum value as is and potentially add a `reason` field on the response. 

Ultimately, what is shown to the end user is a decision that is best made by the application. We can add a `reason` to supplement the response, but is not ideal